src/Structures/NavTree.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace Statamic\Structures;
+
+use Statamic\Events\NavTreeSaved;
+use Statamic\Facades\Site;
+
+class NavTree extends Tree
+{
+    public function path()
+    {
+        $path = base_path('content/structures/navigation/'.$this->handle());
```

Comment:
Maybe this path should be changed to `content/structures/navs` if https://github.com/statamic/cms/pull/3035 gets merged (again for consistency where possible).